src/foundry/foundry.js/applications/formApplications/moduleManagement.d.ts

```diff
@@ -61,17 +51,11 @@ declare class ModuleManagement<
   /** @override */
   activateListeners(html: JQuery): void;
 
-  /**
-   * @override
-   * @param event - (unused)
-   */
-  protected _updateObject(event: Event, formData: ModuleManagement.FormData): Promise<Record<string, boolean>>;
+  /** @override */
+  protected _renderInner(data: Data): Promise<JQuery>;
 
-  /**
-   * Restores the Form UI to the internal checked state
-   * @internal
-   */
-  protected _restoreCheckboxState(): void;
+  /** @override */
+  protected _updateObject(event: Event, formData: ModuleManagement.FormData): Promise<Record<string, boolean>>;
```

Comment:
Shouldn't this also return `Promise<unknown>`?

Reply:
Yea, it should